jumpscale/packages/marketplace/chats/meetings.py

```diff
@@ -0,0 +1,108 @@
+from jumpscale.sals.chatflows.chatflows import chatflow_step
+from jumpscale.sals.marketplace import MarketPlaceAppsChatflow, deployer
+from jumpscale.sals.reservation_chatflow import deployment_context, DeploymentFailed
+
+
+class MeetingsDeploy(MarketPlaceAppsChatflow):
+    FLIST_URL = "https://hub.grid.tf/tobias.3bot/jimbersoftware-freeflowconnect_grid-0.7.flist"
+    SOLUTION_TYPE = "meetings"
+    title = "Meetings"
+    steps = [
+        "get_solution_name",
+        "meetings_info",
+        "infrastructure_setup",
+        "reservation",
+        "initializing",
+        "success",
+    ]
+
+    query = {"cru": 2, "mru": 2, "sru": 0}
```

Comment:
the query is wrong. it doesn't include the nginx container resources which makes the expiration incorrect.
should add 1cru, 1 mru, 0.25 sru, and handle these differences in during i the deployment of the container

Reply:
keep in mind we should distinguish between the total query and the container resources
See: https://github.com/threefoldtech/js-sdk/blob/development_backupcryptpad/jumpscale/packages/tfgrid_solutions/chats/cryptpad_advanced.py#L24

https://github.com/threefoldtech/js-sdk/blob/development_backupcryptpad/jumpscale/packages/tfgrid_solutions/chats/cryptpad_advanced.py#L39

https://github.com/threefoldtech/js-sdk/blob/development_backupcryptpad/jumpscale/packages/tfgrid_solutions/chats/cryptpad_advanced.py#L131